app/services/admin/change_induction_service.rb

```diff
@@ -48,7 +48,7 @@ def perform_induction_change(new_provision)
     end
 
     def withdraw_participants_if_required(new_provision)
-      return if %w[core_induction_programme full_induction_programme].include? new_provision
+      return if %i[core_induction_programme full_induction_programme].include? new_provision
```

Comment:
I sometimes hate Ruby's distinction between strings and symbols, and this is certainly one of those times 😭

Reply:
If we had some kind of type system...